core-primitives/stf-state-handler/src/in_memory_state_file_io.rs

```diff
@@ -204,13 +200,61 @@ where
 	}
 }
 
+#[cfg(feature = "sgx")]
```

Comment:
It's a pity to have to put this as a `sgx` feature only. As far as I can see, only the `list_shards` is sgx only. Is there no way around this?

Reply:
hm yes, maybe I could move more of these functions outside the `sgx` gated module

Reply:
done 👍